src/main/java/com/spotinst/sdkjava/model/bl/ocean/gke/LaunchSpecSpecification.java

```diff
@@ -0,0 +1,360 @@
+package com.spotinst.sdkjava.model.bl.ocean.gke;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class LaunchSpecSpecification {
+
+    @JsonIgnore
+    private Set<String>                                   isSet;
+    private String                                        oceanId;
+    private String                                        id;
```

Comment:
id is not part of api request

Reply:
If I will remove this id from api request then as a response id mayn't get